Faraday_Proxy_Tools/FaradayIO/commandmodule.py

```diff
@@ -301,7 +300,7 @@ def packet_gpio_gps_standby_disable():
 
     .. todo:: This function needs updating to the new create_gpio_command_packet()
     """
-    return create_gpio_command_packet(3, gpio_allocation.GPS_STANDBY, 1)
+    return create_gpio_command_packet(3, gpioallocations.GPS_STANDBY, 1)
```

Comment:
All these changes were made by hand. The module is called `gpioallocations`, I guess this code was never exercised because there is no such variable `gpio_allocations`.